Tests/LibWeb/Text/input/wpt-import/css/css-grid/alignment/grid-alignment-implies-size-change-011.html

```diff
@@ -49,6 +49,6 @@
 </script>
 <body onload="document.fonts.ready.then(() => { runTest(); })">
 <div class="grid" id="grid">
-  <img id="item" src="support/100x100-green.png"></img>
+  <img id="item" src="../../css-flexbox/support/100x100-green.png"></img>
```

Comment:
This isn't the correct URL. The tests on wpt.live are linking to `https://wpt.live/css/css-grid/alignment/support/100x100-green.png`. The real issue seems to be that we don't have that file downloaded.